apps/users/_services/users.service.ts

```diff
@@ -200,7 +202,10 @@ export class UsersService extends BaseService {
         // If user does not have firstTimeSignInAt, it means this is the first time the user is signing in
         // Updates the firstTimeSignInAt with the current date.
         if (!user.firstTimeSignInAt) {
-          await transaction.getRepository(UserEntity).update(user.id, { firstTimeSignInAt: new Date().toISOString() });
+          await transaction.getRepository(UserEntity).update(user.id, {
+            firstTimeSignInAt: new Date().toISOString(),
+            howDidYouFindUsAnswers: data.howDidYouFindUsAnswers
```

Comment:
To avoid saving empty/false values in the database I'd do something like: `Object.fromEntries(Object.entries(data.howDidYouFindUsAnswers).filter(([_k, v]) => v));`